src/main/java/com/project2/demo/Menu/controller/MenuController.java

```diff
@@ -0,0 +1,101 @@
+package com.project2.demo.Menu.controller;
+
+
+import com.project2.demo.Menu.service.MenuService;
+import com.project2.demo.user.service.UserService;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.DateTimeException;
+import java.time.LocalTime;
+import java.time.MonthDay;
+
+@RestController
+@RequestMapping("/menu")
+@AllArgsConstructor
+public class MenuController {
+
+    private final MenuService menuService;
+
+
+    @GetMapping
+    public void createMenu(@RequestBody MenuParameter menuParameter){
+        try {
+            checkTimeValidity(menuParameter);
+            menuService.createMenu(menuParameter);
+        }catch (DateTimeException de){
+            ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Time error");
+        }
+    }
+
+
+    @GetMapping("/{menuId}")
+    public void limitMenu(@PathVariable Long menuId, @RequestBody LimitDayParameter limitDayParameter){
+        try {
+            checkTimeValidity(limitDayParameter);
+            menuService.limitMenu(menuId, limitDayParameter);
+        }catch (DateTimeException de){
+            ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Time error");
+        }catch (IllegalArgumentException argE ){
+            ResponseEntity.status(HttpStatus.BAD_REQUEST);
+        }
+    }
+
+    @DeleteMapping("/menu/{menuId}/delete")
```

Comment:
crud 로 해결가능한 행위라면, uri에 행위를 표시하는 행위는 배제해야합니다.

```suggestion
    @DeleteMapping("/menu/{menuId}")
```

Reply:
지금보니 중복되는 내용도 있네요.
```suggestion
    @DeleteMapping("/{menuId}")
```